src/compiler/cache.ts

```diff
@@ -19,16 +20,18 @@ export class Cache implements d.Cache {
       return;
     }
 
+    this.buildCacheDir = join(this.config.cacheDir, '.build');
```

Comment:
Decided to just make changes here so that the `cacheDir` specified in the Stencil config has a sub-directory for all the build caching files. This prevents us from needing to add or change the config property and eliminates confusion on if that field represented the actual directory where files were written or a parent directory. Also added some JSDoc to the config field to help add clarity

Reply:
makes sense 👍